e2e_tests/visual_regression_test.py

```diff
@@ -1,25 +1,29 @@
-import re
-
-from playwright.sync_api import expect
+import optuna
 from playwright.sync_api import Page
+import pytest
 
 
-def test_homepage_has_Playwright_in_title_and_get_started_link_linking_to_the_intro_page(
+@pytest.mark.parametrize("study_id", range(10))
+def test_study_list(
+    study_id: int,
     page: Page,
+    dummy_storage: optuna.storages.InMemoryStorage,
```

Comment:
I have a question. Is `dummy_storage` created twice per each test since `server` fixture depends on `dummy_storage`.

Reply:
Actually, no if my understanding is correct.
The dummy storage and server fixture are created with `@pytest.fixture(scope="session")`. It means that they are created at the begging of all tests and will be destroyed at the end of the test session.

see more detail in https://docs.pytest.org/en/6.2.x/fixture.html#fixture-scopes